format_spec/group.md

```diff
@@ -33,3 +33,4 @@ The group member is the content inside a [group](./group.md)
 | Relative | `uint8_t` | Is the URI relative to the group |
 | URI length | `uint32_t` | Number of characters in URI |
 | URI | `uint8_t[]` | URI character array |
+| Deleted | `uint8_t` | Is the member deleted |
```

Comment:
So, the only difference between ths v1 and v2 classes is this Deleted field? If so, why not have a single class to handle both format versions? Personally I would do such split if the differences between the versions were more substantial.